### PR TITLE
M13 release - documentation News

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/GettingStarted.rst [hunk 1]
```diff
@@ -1324,7 +1324,7 @@ You can display the pinned articles with one of the following layouts :
 
 |image198|
 
-When you click on the "See all" button, you are redirected automatically to news application filtered by pinned articles.
+When you click on the "See all" button,  you are redirected automatically to news application filtered by pinned articles
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]: The changes on this sentence should be reverted IMO.

File: docs/GettingStarted.rst [hunk 2]
```diff
@@ -1432,23 +1436,73 @@ News application filters
...
 Different filters are available from News application so you can easily find articles posted in specific spaces, pinned articles, your posted articles, your scheduled articles, your drafts or your archived articles.
 
-You can search articles by spaces using the filter button from the top bar.
+
+**Filter articles per spaces : ** You can search articles by spaces using the filter button from the top bar.
 
 |image206|
 
-In the displayed drawer, you can select the spaces of the articles you want to filter. A search input is available to easily find the spaces you want to select. Only the spaces you are member of are available. A click on the Apply button allows to apply the filter.
+In the displayed drawer, you can choose from or search in the list of the spaces in which you are member the space(s) you want to filter by articles in the application then apply the using the button to be redirected to the filtered view.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]: I think you revert the changes we discussed in the previous PR. Is it intended ?

File: docs/GettingStarted.rst [hunk 3]
```diff
@@ -1432,23 +1436,73 @@ News application filters
...
+
+|image209|
+
+After applying this filter, the list will only display the articles posted by you. You have the possibility to archive, edit or share them directly from the application.
+
+|image210|
+
+
+**Filter by archived articles : ** To display your archived articles, select the option "Archived News" from the dropdown menu of the application topbar.
+After applying this filter, the list will only display your archived articles or all archived articles ifyou have the publisher role
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
After applying this filter, the list will only display your archived articles or all archived articles if you have the publisher role
```

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/GettingStarted.rst (hunk 2 above)
Comment (samehaidi) [reply docs/GettingStarted.rst]: I didn't change this part. I created a new branch from master after doing a git fetch git pull. Will check

File: docs/GettingStarted.rst [hunk 4]
```diff
@@ -1410,8 +1410,12 @@ This application is accessible through the ``News`` link  in the left navigation
 
 |image187|
 
-The page displays by default the list of "All News". For each article you can see the title, the illustration, the first words of the summary or the article body, the author, the publication date, the number of views and the information about the source (space).
-You can also share the displayed articles as explained in the `Sharing a News`_ chapter.
+The page displays by default the list of the first 10 articles of the view "All News". You have the possibility to load more articles using the appropriate button.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
The page displays by default the list of the first 10 articles of the view "All News". You can load more articles using the "Load More" button.
```

File: docs/GettingStarted.rst [hunk 5]
```diff
@@ -1410,8 +1410,12 @@ This application is accessible through the ``News`` link  in the left navigation
...
+
+|image217|
+
+For each article you can see the title, the illustration, the first words of the summary or the article body, the author, the publication date, the number of views and the information about the source space as well as the destination space(s) if the article has been shares.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
For each article you can see the title, the illustration, the first words of the summary or the article body, the author, the publication date, the number of views and the original space where the article was posted as well as the spaces where it was shared.
```

File: docs/GettingStarted.rst [hunk 6]
```diff
@@ -1432,23 +1436,73 @@ News application filters
...
 |image197|
 
 After applying this filter, the list will only contain the articles that have been pinned on the home page. You can reset the filters using the appropriate button inside the drawer.
 
 
+**Filter by your posted articles : ** To display the articles written by you, select the option "My posted News" from the dropdown menu of the application topbar.
+
+|image209|
+
+After applying this filter, the list will only display the articles posted by you. You have the possibility to archive, edit or share them directly from the application.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
After applying this filter, the list will only display the articles posted by you. You can either archive, edit or share them directly from the news list.
```

File: docs/GettingStarted.rst (hunk 3 above)
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
After applying this filter, the list will only display your archived articles or all archived articles if you have the publisher role
```

File: docs/GettingStarted.rst [hunk 7]
```diff
@@ -1432,23 +1436,73 @@ News application filters
...
+As an article's author or as a platform-wide publisher (publisher:/platform/web-contributors role required), you can archive articles from News application interface.
+An "archive/unarchive" icon is available in the right side of the article's preview next to share, edit and pin icons.
+
+|image211|
+
+When you click on the archive icon, a confirmation message appears
+
+|image212|
+
+Once you confirm the action, the article is automatically moved into the view "Archived News" of the application and will be no longer available from the application to other users. Only the users having the role publisher (in addition to the author) will be able to see it in their archived view.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Once you confirm the action, the article is automatically moved into the archive and is no longer available to other users. The archive can be accessed from the "Archived News" entry in the filter. Only the authors and publishers can see articles in the archive.
```

File: docs/GettingStarted.rst [hunk 8]
```diff
@@ -1432,23 +1436,73 @@ News application filters
...
+The preview of the archived article activity from the feed will be changed :
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Archived articles have a different display in the activity stream  :
```

File: docs/GettingStarted.rst [hunk 9]
```diff
@@ -1432,23 +1436,73 @@ News application filters
...
+
+|image213|
+
+Other users will not be able to click on this preview as long as the article is archived. Even if they already have the article's URL, the details are no longer visible. The following message appears :
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Other users won't able to click on this preview as long as the article remains archived. Also, for them, accessing an archived article by URL won't let them read it. In such case, the following message appears :
```

File: docs/GettingStarted.rst [hunk 10]
```diff
@@ -1432,23 +1436,73 @@ News application filters
...
+
+|image214|
+
+On the other hand, you still have access to the article's details and you are able to unarchive it from this view.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
However, you, as the author (or publisher), keep have access to the article's details and you can unarchive it from there.
```

File: docs/GettingStarted.rst [hunk 11]
```diff
@@ -1432,23 +1436,73 @@ News application filters
...
+
+|image215|
+
+It is also possible to unarchive articles using the same icon from news application.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
The same icon allows to unarchive articles from the News application.
```

File: docs/GettingStarted.rst [hunk 12]
```diff
@@ -1432,23 +1436,73 @@ News application filters
...
+
+|image216|
+
+Once you confirm the unarchiving action. the article becomes available again to the other users.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
When you confirm the unarchiving of an item, it becomes accessible again to other users.
```

File: docs/GettingStarted.rst [hunk 13]
```diff
@@ -1432,23 +1436,73 @@ News application filters
...
+
+.. note:: When you archive a pinned article it will be automatically unpinned from home page.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
.. note:: When you archive a pinned article it gets automatically unpinned. However, if you unarchive the same article afterwards, it won't be re-pinned automatically. 
```

**Review by plamarque — CHANGES_REQUESTED** [review]
full review and suggestions to smoothen language.

File: docs/GettingStarted.rst (hunk 1 above)
Comment (samehaidi) [reply docs/GettingStarted.rst]: fixed.

File: docs/GettingStarted.rst (hunk 2 above)
Comment (samehaidi) [reply docs/GettingStarted.rst]: Fixed

File: docs/GettingStarted.rst [hunk 14]
```diff
@@ -1410,8 +1410,12 @@ This application is accessible through the ``News`` link  in the left navigation
 
 |image187|
 
-The page displays by default the list of "All News". For each article you can see the title, the illustration, the first words of the summary or the article body, the author, the publication date, the number of views and the information about the source (space).
-You can also share the displayed articles as explained in the `Sharing a News`_ chapter.
+The page displays by default the list of the first 10 articles of the view "All News". You can load more articles using the "Load More" button.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
By default, the page displays the 10 latest articles of the  "All News" list. You can load more articles using the "Load More" button.
```

File: docs/GettingStarted.rst [hunk 15]
```diff
@@ -438,7 +438,8 @@ Quickly perform key actions through the following applications:
 -  *Getting Started*: See a list of suggested actions you can start.
    See :ref:`Getting Started application <Getting Started>` for more details.
 
--  *Calendar*: Get an overview of calendar events. See :ref:`Calendar application <CalendarApp>` for more details.
+-  *Calendar*: Get an overview of calendar events. See :ref:`Calendar application <CalendarApp>
```
Comment (plamarque) [inline docs/GettingStarted.rst]: seems to be unrelated to the news feature

File: docs/GettingStarted.rst [hunk 16]
```diff
@@ -1442,13 +1448,61 @@ In the displayed drawer, you can select the spaces of the articles you want to f
...
-To display the pinned news list, select this option from the dropdown menu of the application topbar.
+
+**Filter by pinned articles : ** To display the pinned news list, select this option from the dropdown menu of the application topbar.
 
 |image197|
 
 After applying this filter, the list will only contain the articles that have been pinned on the home page. You can reset the filters using the appropriate button inside the drawer.
 
 
+**Filter by your posted articles : ** To display the articles written by you, select the option "My posted News" from the dropdown menu of the application topbar.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
**Filter by your posted articles : ** To display the articles written by you, select the option "My posted News" from the dropdown menu of the application's top bar.
```

**Review by thomasdelhomenie — APPROVED** [review]

File: docs/GettingStarted.rst [hunk 17]
```diff
@@ -1442,13 +1448,61 @@ In the displayed drawer, you can select the spaces of the articles you want to f
...
+After applying this filter, the list will only display your archived articles or all archived articles if you have the publisher role
+
+|image218|
+
+
+Archiving News
+---------------
+
+As an article's author or as a platform-wide publisher (publisher:/platform/web-contributors role required), you can archive articles from News application interface.
+An "archive/unarchive" icon is available in the right side of the article's preview next to share, edit and pin icons.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
An "archive/unarchive" icon is available on the right side of the article's preview next to "share", "edit" and "pin" icons.
```

File: docs/GettingStarted.rst [hunk 18]
```diff
@@ -1442,13 +1448,61 @@ In the displayed drawer, you can select the spaces of the articles you want to f
...
+
+|image211|
+
+When you click on the archive icon, a confirmation message appears
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
When you click on the archive icon, a confirmation message appears : 
```

File: docs/GettingStarted.rst [hunk 19]
```diff
@@ -1442,13 +1448,61 @@ In the displayed drawer, you can select the spaces of the articles you want to f
...
+
+|image212|
+
+Once you confirm the action, the article is automatically moved into the archive and is no longer available to other users. The archive can be accessed from the "Archived News" entry in the filter. Only the authors and publishers can see articles in the archive.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Once you confirm the action, the article is automatically moved into the archive and is no longer available to other users. The archive can be accessed from the "Archived News" entry in the dropdown. Only the authors and publishers can see articles in the archive.
```

File: docs/GettingStarted.rst [hunk 20]
```diff
@@ -1442,13 +1448,61 @@ In the displayed drawer, you can select the spaces of the articles you want to f
...
+Archived articles have a different display in the activity stream  :
+
+|image213|
+
+Other users won't able to click on this preview as long as the article remains archived. Also, for them, accessing an archived article by URL won't let them read it. In such case, the following message appears :
+
+|image214|
+
+However, you, as the author (or publisher), keep have access to the article's details and you can unarchive it from there.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
However, you, as the author (or publisher), maintain access to the article's details and you can unarchive it from there.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
review pass #2. Only minor suggestions now.

File: docs/GettingStarted.rst (hunk 15 above)
Comment (samehaidi) [reply docs/GettingStarted.rst]: I didn't update this part.

Comment (samehaidi) [reply docs/GettingStarted.rst]: Fixed

**Review by plamarque — APPROVED** [review]
LGTM
